core/bots.py

```diff
@@ -40,6 +43,25 @@
 LIVE_ROLE_ID: int = 1182206699969458226
 SUBBED_ROLE_ID: int = 873044115279990836
 
+MBTI_TYPES: list[str] = [
```

Comment:
You could probably move this constant into a separate file in the `core` package since it takes up a lot of space in functional code.

E.g. `core/constants.py`
```py
__all__ = ("MBTI_TYPES", )


MBTI_TYPES: list[str] = ...  # Put the list here...
```

And then in `core/__init__.py`
```py
from .constants import *
```

And then in `core/bots.py`

```py
from .constants import WHATEVER_YOU_NEED
```